cypress/server/index.html

```diff
@@ -11,6 +11,10 @@
         var scriptVersion = keepQueryParamIfDefined('version') || '2';
         var scriptUrl =
             keepQueryParamIfDefined('staticurl') ||
+            (keepQueryParamIfDefined('targetEnv').toLowerCase() === 'dev' &&
+                'https://coveo-ndev-binaries.s3.amazonaws.com/proda/StaticCDN/coveo.analytics.js/' +
+                    scriptVersion +
+                    '/coveoua.js') ||
             'https://static.cloud.coveo.com/coveo.analytics.js/' + scriptVersion + '/coveoua.js';
```

Comment:
```suggestion
        var urlPerEnv = {
           "dev": "https://coveo-ndev-binaries.s3.amazonaws.com/proda/StaticCDN/coveo.analytics.js/",
           "prod": "https://static.cloud.coveo.com/coveo.analytics.js/"
        };
        var env = (keepQueryParamIfDefined('targetEnv') || "dev").toLowerCase();
        var scriptUrl =
            keepQueryParamIfDefined('staticurl') || urlPerEnv[env] + scriptVersion + '/coveoua.js';
```

Even if it "wastes" a couple of variables and query param checks, I think it helps for readability

Reply:
Yes I agree, that's much much clearer! I have one question though: When the current environment is the production environment, the `targetEnv` that's passed to the certifier seems to have a value of `prd` and not `prod`, so should `"prod"` be `"prd"` instead?

Reply:
yeah no issue with `prd` :P